templates/oam/workloads/smilinkerdaddon_definition.json

```diff
@@ -0,0 +1,15 @@
+{
+    "apiVersion": "core.oam.dev/v1alpha1",
+    "kind": "WorkloadDefinition",
+    "metadata": {
+        "name": "SMILinkerdAddon"
```

Comment:
Will this be displayed in the UI? If so, should we include whitespace? If not or if this will be computer generated again in the future and be missing the whitespaces then, too, then never mind.

Reply:
As far as I know, this wont be displayed in the UI